src/cli-util.js

```diff
@@ -271,6 +271,8 @@ function eachFilename(argv, patterns, callback) {
       .sync(patterns, { dot: true })
       .map(filePath => path.relative(process.cwd(), filePath));
 
+    filePaths.sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()));
```

Comment:
Just a curious question: Is there a specific reason you use `.localeCompare` instead of just `filePaths.sort()`?

Reply:
@lydell `filePaths.sort()` sorts using case sensitivity (i.e. the same as `filePaths.sort((a, b) => a.localeCompare(b))`

Reply:
Sorry, let me rephrase my question: Is there any specific reason you don’t just use `filePaths.sort()`? Nicer output? Or is it really required to do it case insensitively?

Reply:
The output of `glob` is already sorted. But the case sensitivity is the same for matching (if you want case sensitive matches, it will sort with case sensitivity).

https://github.com/isaacs/node-glob/blob/master/common.js#L153